oasis/access-control.md

```diff
@@ -421,12 +421,21 @@ Deployment Z of project DEF                  | ✓ | — | —
 
 ## Restricting access to organizations
 
-To enhance security, you can limit which authentication providers are accepted for users trying to access an organization in Oasis.
+To enhance security, you can implement the following restrictions via [Oasisctl](oasisctl.html):
 
-To do this, you need to use [Oasisctl](oasisctl.html). The following commands are available:
+1. Limit allowed authentication providers.
+2. Specify an allowed domain list.
```

Comment:
Add note below these options:

Note that users who do not meet the restrictions will not be granted permissions for any resource in
the organization. These users can still be a member of the organization.

Reply:
Done.